demo/package.json

```diff
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "interactive": "suitest interactive mocha --no-timeouts --exit",
+    "debug": "suitest interactive --debug-brk=9121 ./node_modules/.bin/_mocha --no-timeouts --exit",
```

Comment:
why mocha calling for debug script differs from interactive, is this IDE requirements? Which IDE support it?

Reply:
1. for regular run used mocha or ./node_modules/.bin/mocha but for debug one should use _mocha (starting with underscore)
2. To debug used child_process.fork with `execArgv=[--inspect-brk=[port]]` and fork works with node modules not with commands like `mocha`